storage/s3_test.go

```diff
@@ -79,27 +79,6 @@ func TestNewSessionPathStyle(t *testing.T) {
 	}
 }
 
-func TestNewSessionWithRegionSetViaEnv(t *testing.T) {
```

Comment:
why did we remove this test?

Reply:
It was removed following https://github.com/peak/s5cmd/pull/194#discussion_r462100909 

Normally, environment variables are read when region is not provided explicitly. When we run all tests together, sessions created, and thus cached, for one test affects the other. For this test case, since there is already a session with empty/default region created, session is not recreated by reading environment variable; making the test case fail.

Should we add an option to remove cached sessions?

Reply:
When resolving conflicts I undid removal of this test, but we need the [following call](https://github.com/peak/s5cmd/pull/194/files#diff-2c7a8270898140e540697cb13ca5406bR91). We can remove this test case if everyone is ok with it.